content/resources/crownlabs_ssh/crownlabs_ssh.md

```diff
@@ -0,0 +1,114 @@
++++
+fragment = "content"
+#disabled = true
+date = "2021-11-12"
+weight = 100
+# title = "Connecting to CrownLabs with SSH"
++++
+
+## Introduction
+
+**CrownLabs** allows users to access their VMs through a local SSH client (i.e., executed on their local machines).
+
+This guide shows how to generate a SSH key pair on your operating systems (Windows, Linux, MacOS), how to load it on **CrownLabs** and access your **CrownLabs**' VM using SSH from a local terminal.
+
+Note that the **generation** of the **SSH key pair** and the upload of the **SSH public key** does not have to be repeated each time you create a new **VM**.
+This process only needs to be done once per **device.**
+
+## How it works
+
+All CrownLabs VMs are characterized by **private IP addresses**, and cannot be accessed directly through SSH.
+As shown in the figure below, it is necessary to first establish the SSH connection through a **bastion**, which instead is publicly accessible, and then **jump** to the final destination (i.e., the VM).
+
+Through the `ssh -J bastion@crownlabs.polito.it crownlabs@10.1.2.3` command, the local **SSH client** first connects to the bastion (indeed, the **-J** option specifies a **ProxyJump**), leveraging the generated **SSH key-pair** for authentication purposes.
```

Comment:
```suggestion
This can be achieved with the `ssh -J bastion@crownlabs.polito.it crownlabs@10.1.2.3` command, in which the local **SSH client** first connects to the bastion (indeed, the **-J** option specifies a **ProxyJump**), leveraging the generated **SSH key-pair** for authentication purposes.
```